modules/core/core-scheduler/src/main/java/com/enonic/xp/impl/scheduler/ScheduleAuditLogSupportImpl.java

```diff
@@ -149,7 +149,10 @@ private void addResult( final PropertySet targetSet, final ScheduledJob job )
             targetSet.setString( ScheduledJobPropertyNames.USER, job.getUser().toString() );
         }
 
-        targetSet.setString( ScheduledJobPropertyNames.CREATOR, job.getCreator().toString() );
+        if ( job.getCreator() != null )// 7.7 produced jobs with an empty `creator` and `createdTime` fields
```

Comment:
Let's move this comment into test.
Version is 7.7.0 to 7.7.2.

Reply:
But 7.7.0 also did it